src/server/auth/server/roles.go

```diff
@@ -243,11 +244,11 @@ func init() {
 				auth.Permission_CLUSTER_ENTERPRISE_DEACTIVATE,
 				auth.Permission_CLUSTER_DELETE_ALL,
 				auth.Permission_CLUSTER_ENTERPRISE_PAUSE,
-				auth.Permission_PROJECT_DELETE,
 			}),
 	})
 }
 
+// TODO Deduplicate permissiosn in result
 func combinePermissions(permissions ...[]auth.Permission) []auth.Permission {
```

Comment:
I think it would be easy to add the deduplication code here: build up a map, then turn it into a slice and return that.